eventhubs/data-plane/azure-eventhubs-extensions/src/main/java/com/microsoft/azure/eventhubs/extensions/appender/EventHubsManager.java

```diff
@@ -20,7 +20,7 @@
     private EventHubClient eventHubSender;
 
     protected EventHubsManager(final String name, final String eventHubConnectionString) {
-        super(name);
+        super(LoggerContext.getContext(true), name);
```

Comment:
What motivated this change?

Reply:
Eventhubs lib uses log4j-core with 2.5
java-8-parent pom use 2.11.1

Old constructor  updated to 
  protected AbstractManager(LoggerContext loggerContext, String name) {
        this.loggerContext = loggerContext;
        this.name = name;
        LOGGER.debug("Starting {} {}", this.getClass().getSimpleName(), name);
    }

from 
    protected AbstractManager(final String name) {...}